### PR TITLE
Merge changes.md from 3.13.2 release to v3.13-dev

### PR DESCRIPTION
**Pull request by rprouse**
Also updates version to 3.13.3 ready for the next release.

- Update changes for 3.13.2 release
- Fix formatting
- Update version to 3.13.3

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -144,7 +164,7 @@ The .NET Standard 2.0 version of NUnit continues to gain more functionality that
 is found in the .NET 4.5 version of the framework like setting the ApartmentState
 and enabling Timeout on tests.
 
-#### Issues Resolved
+### Issues Resolved
```
Comment (mikkelbu) [inline CHANGES.md:167]: I think that if we start using `###` for `Issues Resolved` and the other sections on the same level, then we should also change all the heading to `##`. E.g. `### NUnit 3.12 - May 14, 2019` => `## NUnit 3.12 - May 14, 2019` (and all the other headers below this one).

Comment (rprouse) [reply CHANGES.md:167]: I didn't even look at these older ones so I missed that.

File: CHANGES.md [hunk 2]
```diff
@@ -1035,28 +1055,28 @@ use the NUnit NuGet packages for the framework, but a ZIP file with the binaries
...
-#### Console Runner
+### Console Runner
 
  * The console now displays all settings used by the engine to run tests as well as the filter used to select tests.
  * The console runner accepts a new option --maxagents. If multiple assemblies are run in separate processes, this value may be used to limit the number that are executed simultaneously in parallel.
  * The console runner no longer accepts the --include and --exclude options. Instead, the new --where option provides a more general way to express which tests will be executed, such as --where "cat==Fast && Priority==High". See the docs for details of the syntax.
  * The new --debug option causes NUnit to break in the debugger immediately before tests are run. This simplifies debugging, especially when the test is run in a separate process.
 
-##### Issues Resolved
+#### Issues Resolved
```
Comment (mikkelbu) [inline CHANGES.md:1079]:
```suggestion
### Issues Resolved
```

**Review by mikkelbu — APPROVED** [review]
One last comment :)

**Pull request merged**